pkgs/development/ocaml-modules/kappa-server/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildDunePackage
+, num
+, re
+, yojson
+, lwt
+, fmt
+, logs
+, kappa-binaries
+, kappa-agents
+, cohttp-lwt-unix
+, atdgen-runtime
+, atdgen
+}:
+
+buildDunePackage {
+  pname = "kappa-server";
+  inherit (kappa-binaries) version src;
+
+  nativeBuildInputs = [
+    atdgen
+  ];
+
+  propagatedBuildInputs = [
```

```suggestion
  buildInputs = [
```

This derivation only produces a binary. The OCaml libraries used for building it need not be propagated.